Dockerfile

```diff
@@ -17,7 +18,26 @@ RUN buildDeps='libmagic-dev' && \
     rm -rf /var/lib/apt/lists/* && \
     bundle install --deployment --without development test && \
     apt-get purge -y --auto-remove $buildDeps && \
-    whenever --update-crontab
+    echo "Foodsoft::Application.config.secret_token = ENV['SECRET_KEY_BASE']" > config/initializers/secret_token.rb && \
```

Comment:
I don't want to break any existing deployment. If we really want to change it (with all the documentation updates) we should switch to [secrets.yml](http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#config-secrets-yml), which is more work than just getting the Docker setup running...